### PR TITLE
migrating router-bgp-peer to be a handwritten resource

### PR DESCRIPTION
**Pull request by Gorlami96**
Moving the resource router-bgp-peer to the handwritten section because we want to add md5 authentication keys as a new field which is not possible right now with the current auto-generated code. Related to b/260899743

**Release Notes**

```release-note:none
Migrating resource from auto-generated to handwritten section
```

### CODE REVIEW

File: mmv1/third_party/terraform/services/compute/resource_compute_router_peer_test.go.erb [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+
+	<% if version == "ga" -%>
```
Comment (trodge) [inline mmv1/third_party/terraform/services/compute/resource_compute_router_peer_test.go.erb:16]: This file doesn't actually use the compute go library. When it was auto-generated the generator ran goimports on it removing unnecessary imports.

To address this I would remove L16-20 here altogether.

**Review by trodge — CHANGES_REQUESTED** [review]
Looks like there are errors in the build related to imports:
```
Error: ../../../go/pkg/mod/github.com/modular-magician/terraform-provider-google-beta@v1.20.1-0.20231204071049-9b4179b261ae/google-beta/services/compute/resource_compute_router_peer.go:21:2: "google.golang.org/api/compute/v0.beta" imported as compute and not used
make: *** [Makefile:6: build] Error 1
Error: Process completed with exit code 2.
```

File: mmv1/third_party/terraform/services/compute/resource_compute_router_peer_test.go.erb (hunk 1 above)
Comment (Gorlami96) [reply mmv1/third_party/terraform/services/compute/resource_compute_router_peer_test.go.erb]: Removed and pushed the change

**Comment by Gorlami96** [comment]
Removed the same lines of code from resource_compute_router_peer.go.erb as well

**Pull request merged**